src/assets/yaml/questions.yml

```
@@ -0,0 +1,138 @@
+---
+- letter: A
+  number: 1
+  questions:
+  - 15 / 3 - 4
+  - (2 + 9) / 11
+  - 63 / 7 / 9
+  - 3 - 11 + 9
+  - 19 - 3 * 6
+  - 2 * 5 - 9
+  - 21 - 12 - 8
+  - 80 / 10 - 7
+- letter: D
+  number: 4
+  questions:
+  - 18 / 6 + 1
+  - (12 + 28) / 10
+  - 7 * 2 - 10
+  - 23 - 15 - 4
+- letter: E
+  number: 5
+  questions:
+  - 24 / 3 - 3
+  - 4 * 5 - 15
+  - 60 / 3 / 4
+  - 4 - 11 + 12
+  - (32 - 31) * 5
+  - 44 - 3 * 13
+  - (23 + 2) / 5
+  - 10 / 2 + 0
+- letter: G
+  number: 7
+  questions:
+  - 3 - 7 + 11
+  - 55 / 11 + 2
+  - 15 - 2 * 4
+  - (41 - 6) / 5
+- letter: H
+  number: 8
+  questions:
+  - 9 / 3 + 5
+  - (32 + 8) / 5
+  - 5 + 12 - 9
+  - 5 * 6 - 22
+  - 36 / 9 * 2
+  - (52 - 20) / 4
+  - 29 - 3 * 7
+  - 26 - 9 - 9
+- letter: I
+  number: 9
+  questions:
+  - 2 - 8 + 15
+  - (11 - 8) * 3
+  - 32 / 4 + 1
+  - 36 / 2 / 2
+  - 25 - 2 * 8
+  - 31 - 12 - 10
+  - (13 + 14) / 3
+  - 2 * 4 + 1
+- letter: L
+  number: 12
+  questions:
+  - 42 / 6 + 5
+  - 7 - 15 + 20
+  - 5 * 2 + 2
+  - 27 - 5 * 3
+- letter: N
+  number: 14
+  questions:
+  - 41 - 20 - 7
+  - 22 - 4 * 2
+  - 60 / 5 + 2
+  - 21 / 3 * 2
+  - 21 + 9 - 16
+  - 34 - 4 * 5
+  - 2 * 4 + 6
+  - (31 - 3) / 2
+- letter: O
+  number: 15
+  questions:
+  - 3 * 3 + 6
+  - 14 - 15 + 16
+  - 44 / 4 + 4
+  - 37 - 11 * 2
+- letter: P
+  number: 16
+  questions:
+  - 43 - 22 - 5
+  - 25 / 5 + 11
+  - 14 / 7 * 8
+  - 1 + 5 * 3
+  - 69 / 3 - 7
+  - 6 * 5 - 14
+  - (41 + 7) / 3
+  - 23 - 18 + 11
+- letter: S
+  number: 19
+  questions:
+  - 42 / 3 + 5
+  - 11 * 3 - 14
+  - (37 + 1) / 2
+  - 25 - 18 / 3
+- letter: T
+  number: 20
+  questions:
+  - 40 / 5 + 12
+  - 25 * 4 / 5
+  - 51 - 14 - 17
+  - 42 - 11 * 2
+  - 3 * 4 + 8
+  - (23 + 17) / 2
+  - 80 / (11 - 7)
+  - 2 - 15 + 33
+- letter: U
+  number: 21
+  questions:
+  - 31 - 3 - 7
+  - 54 / 3 + 3
+  - 15 + 2 * 3
+  - 8 * 4 - 11
+- letter: V
+  number: 22
+  questions:
+  - (4 + 7) * 4
```

@kartikpranav this does not equal to 22
```suggestion
  - (4 + 7) * 2
```